aibolit/patterns/er_class/er_class.py

```diff
@@ -1,14 +1,15 @@
-import javalang
-
+from aibolit.ast_framework import ASTNodeType, AST
```

Comment:
Add header with license. I am using 'File Header Comment' extension to simplify this task.

Reply:
added